python/eos/tasks.py

```diff
@@ -100,10 +100,28 @@ def find_clusters(posterior, base_directory='./', threshold=2.0, K_g=1, analysis
     eos.info(f'Created mixture density with {len(density.components)} components')
     eos.data.MixtureDensity.create(output_path, density)
 
+def mixture_product(posterior, posteriors, base_directory='./', analysis_file=None):
+    """
+    Compute the cartesian product of the densities listed in posteriors. Note that this product is not commutative.
+
+    The input densities are read from EOS_BASE_DIRECTORY/POSTERIOR_i/pmc, where POSTERIOR_i is listed in posteriors.
+    The output density will be stored in EOS_BASE_DIRECTORY/POSTERIOR/product.
+
+    :param posterior: The name of the posterior.
+    :type posterior: str
+    :param posteriors: The list of names of the posteriors whose mixture densities will be concatenated.
+    :type posteriors: iterable of str
```

Comment:
``posterior`` vs ``posteriors`` is likely to cause trouble, but I have not good (short!) alternative names.

Reply:
This should not affect the user since those are mandatory arguments